docs/references/development_workflow.rst

```diff
@@ -82,3 +84,78 @@ Within the learning equality repository:
 If a change needs to be introduced to an older release, target the oldest release branch that we want the change made in. Then that change will need to be merged into all subsequent releases, one-at-a-time, until it eventually gets back to ``develop``.
 
 
+Github Labels
+-------------
+
+We use a `wide range of labels <https://github.com/learningequality/kolibri/labels>`__ to help organize issues and pull requests in the Kolibri repo.
+
+Changelog
+~~~~~~~~~
+
+The **changelog** label is used on PRs or issues to generate 'more details' links in the :ref:`changelog`.
+
+
+PR Lifecycle
+~~~~~~~~~~~~
+
+These labels help coordinate code reviews and our pull-request workflow. A PR should always have exactly one of these labels:
+
+* **PR: needs review**
+* **PR: needs updates**
+* **PR: work-in-progress**
+
+
+Development Category
+~~~~~~~~~~~~~~~~~~~~
+
+Used to help organize issues and sometimes PRs by area of responsibility or scope of domain knowledge necessary. They are not mutually exclusive.
+
+* **DEV: backend**
+* **DEV: content renderers**
+* **DEV: content**
+* **DEV: dev-ops**
+* **DEV: distributions**
+* **DEV: frontend**
+* **DEV: performance**
+
+
+Priority
+~~~~~~~~
+
+These are used to sort issues and sometimes PRs by priority if *and only if* the item is assigned a milestone. Every issue in a milestone ought to have a priority label.
+
+Generally, only 'critical' items are considered blockers for a release.
+
+* **P0 - critical**
+* **P1 - important**
+* **P2 - normal**
+* **P3 - low**
+
+
+Tasks
+~~~~~
+
+These labels are generally used to categorize issues by the *outcome* of closing the issue. They are not necessarily mutually exclusive, but often are.
```

Comment:
I have never understood these labels clearly, the description have unfortunately not cleared it up. Rather than using labels, shouldn't tasks always be expressed by the opening of new issues?

Reply:
It's still worded poorly then. It's supposed to indicate "task type or category", and in particular what effect or outcome it will have. Does that make more sense?